contracts/Lockbox.cdc

```diff
@@ -0,0 +1,329 @@
+import FlowToken from 0x0ae53cb6e3f42a79
+import FungibleToken from 0xee82856bf20e2aa6
+import FlowIDTableStaking from 0x179b6b1cb6755e31
+
+pub contract Lockbox {
+
+    // 1. Admin: Dapper Labs only
+    // 2. Shared: Dapper Labs + Token Holder
+    // 3. Holder: Token Holder only
+
+    /// path to store the locked token manager resource 
+    /// in the shared account
+    pub let LockedTokenManagerPath: Path
+
+    /// path to store the private locked token admin link
+    /// in the shared account
+    pub let LockedTokenAdminPrivatePath: Path
+
+    /// path to store the admin collection 
+    /// in the admin account
+    pub let LockedTokenAdminCollectionPath: Path
+
+    /// path to store the token holder resource
+    /// in the unlocked account
+    pub let TokenHolderStoragePath: Path
+
+    /// path to store the node operator struct
+    /// in the node operators account for staking helper
+    pub let NodeOperatorCapabilityStoragePath: Path
```

Comment:
I wonder if this should be defined here?

Reply:
Probably not. I'll define it in the StakingProxy contract